test/e2e/lib/gutenberg/blocks/tiled-gallery-block-component.js

```diff
@@ -13,12 +13,17 @@ class TiledGalleryBlockComponent extends GutenbergBlockComponent {
 	static blockName = 'jetpack/tiled-gallery';
 	static blockFrontendSelector = By.css( '.entry-content .wp-block-jetpack-tiled-gallery' );
 
-	fileInputSelector = By.xpath(
-		`//*[@id="${ this.blockID.slice( 1 ) }"]/div/div/div[3]/div[2]/input`
-	);
-
+	/**
+	 * Uploads images to the gallery.
+	 *
+	 * @param {{imageName: string, fileName: string, file: string}} filesDetails a list of fileDetails
+	 */
 	async uploadImages( filesDetails ) {
-		const fileInput = this.driver.findElement( this.fileInputSelector );
+		const fileInputSelector = By.xpath(
+			`//*[@id="${ this.blockID.slice( 1 ) }"]/div/div/div[3]/div[2]/input`
```

Comment:
Can we use the same selector as with the Masonry gallery? This one looks very fragile, especially this part: `/div/div/div[3]/div[2]/`.

Reply:
Yes, we can 👍🏻 I actually forgot to change this one.